scripts/validate-seams.ts

```
@@ -0,0 +1,62 @@
+    }
+  });
+}
+
+console.log('🔍 Starting Seam Validation...');
+
+// Walk source
+walkDir('src', scanFile);
+
+if (VIOLATIONS.length > 0) {
+  console.error('❌ Seam Validation FAILED:');
+  VIOLATIONS.forEach(v => console.error(v));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)